app/containers/message/Urls.tsx

```diff
@@ -48,21 +48,51 @@ const styles = StyleSheet.create({
 		height: 150,
 		borderTopLeftRadius: 4,
 		borderTopRightRadius: 4
+	},
+	imageWithoutContent: {
+		borderRadius: 4
+	},
+	activityPosition: {
+		zIndex: 2,
+		position: 'absolute'
+	},
+	loadingState: {
+		height: 0,
+		borderWidth: 0
```

Comment:
I refactor the whole PR to add the possibility of showing nothing during loading, instead appears something loading and then disappears because of an error from  onError function

https://user-images.githubusercontent.com/47038980/219213894-de4ef541-52bc-4b0e-8300-a00f76871387.mov

Reply:
@reinaldonetof I don't understand what I'm supposed to be looking for on this 3s video.

Reply:
before this refactoring of the component, during the loading state, the `Touchable` (https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4723/files/249a056b7dd0a3b991be58e87b7ed14eb08d7952#diff-b1ec2afd1adf1d4430ea9e5090e7107f68f598bc493ca059ba5b38dade0c97fbR146) was rendering a blank background and was affecting the UX. Now I hide this Touchable while it's loading